packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -162,12 +162,12 @@ export class TextField {
   @Prop() max: string | number = undefined;
 
   /**
-   * The maximum number of characters that can be entered in the field.
+   * The maximum number of characters that can be entered in the field. Will emit an error if too many characters are entered.
```

Comment:
I feel like "display" might be more appropriate as "emit" is generally associated with events
```suggestion
   * The maximum number of characters that can be entered in the field. Will display an error if too many characters are entered.
```